packages/components/src/font-size-picker/index.tsx

```diff
@@ -123,6 +124,12 @@ const UnforwardedFontSizePicker = (
 		!! valueUnit && [ 'em', 'rem', 'vw', 'vh' ].includes( valueUnit );
 	const isDisabled = value === undefined;
 
+	maybeWarnDeprecated36pxSize( {
+		componentName: 'FontSizePicker',
+		__next40pxDefaultSize,
+		size: undefined,
```

Comment:
This component does have a `size` prop so it needs to be passed.

```suggestion
		size,
```